### ISSUE TITLE
r9 280x 6gb tahiti win64 version stuck in eth mining

### ISSUE THREAD

**Issue by gpu7990** [Lolliedieb/lolMiner-releases#889]
When i start etc- all works fine, but in eth mode machine totally stuck after DAG creation complete, then need reboot.
Thanks!

**Comment by jgonzis** [comment]
> When i start etc- all works fine, but in eth mode machine totally stuck after DAG creation complete, then need reboot. Thanks!

Have you got enough Virtual Disk?

**Comment by gpu7990** [comment]
Hi, 48gb, i think problem only in lolminer(kernel) because etc works fine. 280x 6gb it is rare card, most of them are 3gb

**Comment by gpu7990** [comment]
Hello, any info? i have also s10000 12gb cards and they also works only in linux, can it be fixed?

**Comment by jgonzis** [comment]
> Hello, any info? i have also s10000 12gb cards and they also works only in linux, can it be fixed?

Try older drivers if helps...

**Comment by gpu7990** [comment]
You are right, everything works on the old 17.4 driver, but the speed is only 8mhash. Compute mode enabled /
18 19 20 21 drivers does not work.

**Comment by jgonzis** [comment]
> You are right, everything works on the old 17.4 driver, but the speed is only 8mhash. Compute mode enabled / 18 19 20 21 drivers does not work.

It seems like it wasn't applyed the compute mode... try with ETCHASH, or give a try in Linux to see what happens, with MMPOS or HiveOS

**Comment by gpu7990** [comment]
In ETC i have 10Mh/
On hiveOS i have 26.3Mh, but the problem i that base mem clock in driver set to 1500Mhz and  i can not control voltage, gpu and mem clock. Also i can not flash vbios on Tahiti cards/
All these options available on WIN10 and these cards works on 1900Mhz mem

**Comment by SomethingGettingWrong** [comment]
listen this is slightly off topic but somewhat the same. I bought an s9300x2 which is a fiji core (radeon pro equivilent) its very rare server version of the radoen pro.. but has the same gpu as NANO, Fury,Furyx...  

teh s9300x2 is not supported on blockchain drivers. so i used 20.11.2 ( which is the best single threaded driver that supports the widest range of hardware) later drivers handicapp older cards.    

Well 20.11.2 picksit up as s9300x2 and it did 15 etc  and 7 beamv3 sol/s  and only pulled 70 watts on 2 gpus on the sambe board 35 watts each... 

Well i flashed to a Radeon Pro bios and then the control panel turned blue. I then  was able to "make compute mode" cut on.. and it was a RADEON PRO now...  however.. it legit.. was still 15 megahash on etc and 7  on beamv3... 

So i flashed to NANO bios.. and blockchain picked up both gpu's as 2 discrete cards.... with seperate controls.. i enabled compute mode (which bluescreened but enabled) reset the pc and now can mine 46 sol/s on s9300x2 as wellas 55 megahash on etc on the card.    

So my point is that AMD on rare cards some drivers dont pick them up even though they should.... even though they are same hardware... with the nano bios flashed on each gpu on one card i am at full tdp if need to..  where as with the s9300x2 bios it just woudlnt detect or run at wrong hash. 

So im not sure if there are diffrent "versions" of this card  but I CAN ASSURE YOU your card isnt running in compute mode 

Why cant you flash vbios on Tahiti cards?

**Comment by SomethingGettingWrong** [comment]
![image](https://user-images.githubusercontent.com/36722911/163658985-7abe6448-e72b-4f74-9d13-1a2c131d1552.png)
I use normal adrenaline on a server GPU lol... with nano bios. so try modding the bios to unlock performance in windows. 

Device Type:                                   CL_DEVICE_TYPE_GPU
  Vendor ID:                                     1002h
  Board name:                                    AMD Radeon (TM) R9 Fury Series
  Device Topology:                               PCI[ B#5, D#0, F#0 ]
  Max compute units:                             64
  Max work items dimensions:                     3
    Max work items[0]:                           1024
    Max work items[1]:                           1024
    Max work items[2]:                           1024
  Max work group size:                           256
  Preferred vector width char:                   4
  Preferred vector width short:                  2
  Preferred vector width int:                    1
  Preferred vector width long:                   1
  Preferred vector width float:                  1
  Preferred vector width double:                 1
  Native vector width char:                      4
  Native vector width short:                     2
  Native vector width int:                       1
  Native vector width long:                      1
  Native vector width float:                     1
  Native vector width double:                    1
  Max clock frequency:                           880Mhz
  Address bits:                                  64
  Max memory allocation:                         4026531840
  Image support:                                 Yes
  Max number of images read arguments:           128
  Max number of images write arguments:          64
  Max image 2D width:                            16384
  Max image 2D height:                           16384
  Max image 3D width:                            2048
  Max image 3D height:                           2048
  Max image 3D depth:                            2048
  Max samplers within kernel:                    16
  Max size of kernel argument:                   1024
  Alignment (bits) of base address:              2048
  Minimum alignment (bytes) for any datatype:    128
  Single precision floating point capability
    Denorms:                                     No
    Quiet NaNs:                                  Yes
    Round to nearest even:                       Yes
    Round to zero:                               Yes
    Round to +ve and infinity:                   Yes
    IEEE754-2008 fused multiply-add:             Yes
  Cache type:                                    Read/Write
  Cache line size:                               64
  Cache size:                                    16384
  Global memory size:                            4294967296
  Constant buffer size:                          4026531840
  Max number of constant args:                   8
  Local memory type:                             Scratchpad
  Local memory size:                             32768
  Max pipe arguments:                            16
  Max pipe active reservations:                  16
  Max pipe packet size:                          4026531840
  Max global variable size:                      3623878656
  Max global variable preferred total size:      4294967296
  Max read/write image args:                     64
  Max on device events:                          1024
  Queue on device max size:                      8388608
  Max on device queues:                          1
  Queue on device preferred size:                262144
  SVM capabilities:
    Coarse grain buffer:                         Yes
    Fine grain buffer:                           Yes
    Fine grain system:                           No
    Atomics:                                     No
  Preferred platform atomic alignment:           0
  Preferred global atomic alignment:             0
  Preferred local atomic alignment:              0
  Kernel Preferred work group size multiple:     64
  Error correction support:                      0
  Unified memory for Host and Device:            0
  Profiling timer resolution:                    1
  Device endianess:                              Little
  Available:                                     Yes
  Compiler available:                            Yes
  Execution capabilities:
    Execute OpenCL kernels:                      Yes
    Execute native function:                     No
  Queue on Host properties:
    Out-of-Order:                                No
    Profiling :                                  Yes
  Queue on Device properties:
    Out-of-Order:                                Yes
    Profiling :                                  Yes
  Platform ID:                                   00007FFECC1E9490
  Name:                                          Fiji
  Vendor:                                        Advanced Micro Devices, Inc.
  Device OpenCL C version:                       OpenCL C 2.0
  Driver version:                                3188.4
  Profile:                                       FULL_PROFILE
  Version:                                       OpenCL 2.0 AMD-APP (3188.4)
  Extensions:                                    cl_khr_fp64 cl_amd_fp64 cl_khr_global_int32_base_atomics cl_khr_global_int32_extended_atomics cl_khr_local_int32_base_atomics cl_khr_local_int32_extended_atomics cl_khr_int64_base_atomics cl_khr_int64_extended_atomics cl_khr_3d_image_writes cl_khr_byte_addressable_store cl_khr_fp16 cl_khr_gl_sharing cl_khr_gl_depth_images cl_amd_device_attribute_query cl_amd_vec3 cl_amd_printf cl_amd_media_ops cl_amd_media_ops2 cl_amd_popcnt cl_khr_d3d10_sharing cl_khr_d3d11_sharing cl_khr_dx9_media_sharing cl_khr_image2d_from_buffer cl_khr_spir cl_khr_subgroups cl_khr_gl_event cl_khr_depth_images cl_khr_mipmap_image cl_khr_mipmap_image_writes cl_amd_liquid_flash cl_amd_planar_yuv

**Comment by jgonzis** [comment]
Nice comments
